src/Domain/CoordinateMaps/CoordinateMap.hpp

```diff
@@ -497,10 +529,6 @@ class CoordinateMap
                    Maps...>());
     }
   }
-
-  /// Compute the inverse Hessian of the `Maps...` at the point(s)
-  /// `source_point` by computing the derivatives of the inverse
-  /// Jacobian of the `Maps...`.
   InverseHessian<DataVector, dim, SourceFrame, TargetFrame> inv_hessian(
       tnsr::I<DataVector, dim, SourceFrame> source_point,
```

Comment:
I can't actually select the line below, but would you mind adding a commit that changes the `coords_frame_velocity_jacobians` functions to trailing return type syntax (`auto coords_frame_velocity_jacobians(...) -> std::tuple<...>`)? That will make the dox render a lot nicer for this class